.github/workflows/build_tao_community_ami.yaml

```diff
@@ -0,0 +1,18 @@
+name: Update taocloud-ami-luoss on new TAO Community release
+on:
+  create:
+    # Trigger the action only when a new tag is created
+    tags:
+      - '*'
+
+jobs:
+  build-tao-community-ami:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send repository_dispatch event
+        uses: peter-evans/repository-dispatch@v1
+        with:
+          token: ${{ secrets.CI_GITHUB_TOKEN }}
+          repository: oat-sa/taocloud-ami-luoss
+          event-type: build-tao-community-ami
+          client-payload: '{"tag": "${{ github.ref }}", "branch": "build-tao-community"}'
```

Comment:
`ref_name` could be a better fit, considering we run the whole workflow on a tag push.
Or is the full name required here for some reason?

Reply:
Also, why is the branch name hardcoded here?

Reply:
according to [documentation](https://github.com/peter-evans/repository-dispatch#:~:text=%3A%20my%2Devent-,Action%20inputs,-Name) we need a full name, the branch is hardcoded as it will be useful to understand where the code for the AMI build is located

Reply:
now I got what you meant, thanks